src/SprykerEco/Zed/Payone/PayoneDependencyProvider.php

```diff
@@ -25,6 +25,11 @@ class PayoneDependencyProvider extends AbstractBundleDependencyProvider
     public const FACADE_GLOSSARY = 'glossary facade';
     public const FACADE_CALCULATION = 'calculation facade';
 
+    /**
+     * @uses \Spryker\Yves\Http\Plugin\Application\HttpApplicationPlugin::SERVICE_REQUEST_STACK
```

Comment:
It's a Zed application use the following link:
* `\Spryker\Zed\Http\Communication\Plugin\Application\HttpApplicationPlugin::SERVICE_REQUEST_STACK`.

Reply:
Thank you